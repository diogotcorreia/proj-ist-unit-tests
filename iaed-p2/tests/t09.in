set /70M !2(
set /70M/adS _3)
set /70M/adS/ypl =M'
set /70M/adS/ypl/iQQ DT_
set /70M/adS/ypl/Ukn 6^b
set /70M/adS/ypl/ptU wNR
set /70M/adS/7TG EFx
set /70M/adS/7TG/IL3 #p@
set /70M/adS/7TG/XyG H&*
set /70M/adS/7TG/Rvr ~w|
set /70M/adS/dsC .i^
set /70M/adS/dsC/thG r\$
set /70M/adS/dsC/YVE xsZ
set /70M/adS/dsC/PgV hK=
set /70M/0Pp e\4
set /70M/0Pp/yG4 #Pu
set /70M/0Pp/yG4/v_M =43
set /70M/0Pp/yG4/2VN /dS
set /70M/0Pp/yG4/GeS KaG
set /70M/0Pp/MPq 2.d
set /70M/0Pp/MPq/WJ_ rbU
set /70M/0Pp/MPq/y32 {S%
set /70M/0Pp/MPq/I23 Wdl
set /70M/0Pp/k5W |Xg
set /70M/0Pp/k5W/iRC 7Rg
set /70M/0Pp/k5W/N_z oAZ
set /70M/0Pp/k5W/d2n EZv
set /70M/mo8 wii
set /70M/mo8/xn6 l?&
set /70M/mo8/xn6/Ykw hS'
set /70M/mo8/xn6/3MC +PV
set /70M/mo8/xn6/o38 (0}
set /70M/mo8/XMA 9'A
set /70M/mo8/XMA/YBm U)%
set /70M/mo8/XMA/PUv V8H
set /70M/mo8/XMA/Akj >D6
set /70M/mo8/g8P e)'
set /70M/mo8/g8P/Lmz a_A
set /70M/mo8/g8P/z4o AY^
set /70M/mo8/g8P/jtt &%8
set /b_h :RE
set /b_h/Lhx x+*
set /b_h/Lhx/yCN R}q
set /b_h/Lhx/yCN/RXe qVp
set /b_h/Lhx/yCN/MQC W$b
set /b_h/Lhx/yCN/7lX Bw6
set /b_h/Lhx/o_n [Q>
set /b_h/Lhx/o_n/Wt9 1r/
set /b_h/Lhx/o_n/15c c`w
set /b_h/Lhx/o_n/nf8 6;g
set /b_h/Lhx/Mw_ 2=[
set /b_h/Lhx/Mw_/qJI ?xI
set /b_h/Lhx/Mw_/Fnb GVp
set /b_h/Lhx/Mw_/Y79 8*n
set /b_h/JTk &gH
set /b_h/JTk/cUW d`h
set /b_h/JTk/cUW/7r7 %v>
set /b_h/JTk/cUW/Mlj Dgo
set /b_h/JTk/cUW/Le5 re[
set /b_h/JTk/GLC ];*
set /b_h/JTk/GLC/KwL ;2X
set /b_h/JTk/GLC/94G uGt
set /b_h/JTk/GLC/nHn "7|
set /b_h/JTk/ba0 y,]
set /b_h/JTk/ba0/qoU Tm[
set /b_h/JTk/ba0/RKr mNs
set /b_h/JTk/ba0/Cbn 3V{
set /b_h/BOs $Qh
set /b_h/BOs/oBC 15f
set /b_h/BOs/oBC/5t_ /N@
set /b_h/BOs/oBC/DwS K9J
set /b_h/BOs/oBC/RSp 2Y{
set /b_h/BOs/TPq [b}
set /b_h/BOs/TPq/jX2 @=D
set /b_h/BOs/TPq/nN0 0)t
set /b_h/BOs/TPq/qLs J {
set /b_h/BOs/RKv p7e
set /b_h/BOs/RKv/8jF O~d
set /b_h/BOs/RKv/ZDB >a<
set /b_h/BOs/RKv/pCN !|O
set /zgN E0X
set /zgN/HJf }a*
set /zgN/HJf/UHb j7P
set /zgN/HJf/UHb/CWR xoI
set /zgN/HJf/UHb/cUl 5EA
set /zgN/HJf/UHb/6eu %Tq
set /zgN/HJf/wFU tnt
set /zgN/HJf/wFU/q_q &zf
set /zgN/HJf/wFU/wbK w|+
set /zgN/HJf/wFU/4P8 wqx
set /zgN/HJf/dzx ~,<
set /zgN/HJf/dzx/8V2 }/z
set /zgN/HJf/dzx/xgo XY3
set /zgN/HJf/dzx/NEB !#\
set /zgN/AFK ]|5
set /zgN/AFK/M4K M'J
set /zgN/AFK/M4K/Hn3 T@%
set /zgN/AFK/M4K/YlE D`r
set /zgN/AFK/M4K/j1e 8o$
set /zgN/AFK/aXE XgQ
set /zgN/AFK/aXE/jBv d&@
set /zgN/AFK/aXE/Bad 4)D
set /zgN/AFK/aXE/jDv L1q
set /zgN/AFK/33m ?6E
set /zgN/AFK/33m/iBj 0>/
set /zgN/AFK/33m/hbg f3R
set /zgN/AFK/33m/Ls3 _et
set /zgN/y7l NQr
set /zgN/y7l/3Eq 1 8
set /zgN/y7l/3Eq/9KV *L=
set /zgN/y7l/3Eq/UDg Pm*
set /zgN/y7l/3Eq/_z7 )XL
set /zgN/y7l/roY wIR
set /zgN/y7l/roY/xEX =@#
set /zgN/y7l/roY/YzW `?J
set /zgN/y7l/roY/CQL OV@
set /zgN/y7l/tZu DS\
set /zgN/y7l/tZu/T_Q }A"
set /zgN/y7l/tZu/ivU ^i/
set /zgN/y7l/tZu/p3Z "?H
print
quit
