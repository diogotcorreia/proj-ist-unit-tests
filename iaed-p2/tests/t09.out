/70M !2(
/70M/adS _3)
/70M/adS/ypl =M'
/70M/adS/ypl/iQQ DT_
/70M/adS/ypl/Ukn 6^b
/70M/adS/ypl/ptU wNR
/70M/adS/7TG EFx
/70M/adS/7TG/IL3 #p@
/70M/adS/7TG/XyG H&*
/70M/adS/7TG/Rvr ~w|
/70M/adS/dsC .i^
/70M/adS/dsC/thG r\$
/70M/adS/dsC/YVE xsZ
/70M/adS/dsC/PgV hK=
/70M/0Pp e\4
/70M/0Pp/yG4 #Pu
/70M/0Pp/yG4/v_M =43
/70M/0Pp/yG4/2VN /dS
/70M/0Pp/yG4/GeS KaG
/70M/0Pp/MPq 2.d
/70M/0Pp/MPq/WJ_ rbU
/70M/0Pp/MPq/y32 {S%
/70M/0Pp/MPq/I23 Wdl
/70M/0Pp/k5W |Xg
/70M/0Pp/k5W/iRC 7Rg
/70M/0Pp/k5W/N_z oAZ
/70M/0Pp/k5W/d2n EZv
/70M/mo8 wii
/70M/mo8/xn6 l?&
/70M/mo8/xn6/Ykw hS'
/70M/mo8/xn6/3MC +PV
/70M/mo8/xn6/o38 (0}
/70M/mo8/XMA 9'A
/70M/mo8/XMA/YBm U)%
/70M/mo8/XMA/PUv V8H
/70M/mo8/XMA/Akj >D6
/70M/mo8/g8P e)'
/70M/mo8/g8P/Lmz a_A
/70M/mo8/g8P/z4o AY^
/70M/mo8/g8P/jtt &%8
/b_h :RE
/b_h/Lhx x+*
/b_h/Lhx/yCN R}q
/b_h/Lhx/yCN/RXe qVp
/b_h/Lhx/yCN/MQC W$b
/b_h/Lhx/yCN/7lX Bw6
/b_h/Lhx/o_n [Q>
/b_h/Lhx/o_n/Wt9 1r/
/b_h/Lhx/o_n/15c c`w
/b_h/Lhx/o_n/nf8 6;g
/b_h/Lhx/Mw_ 2=[
/b_h/Lhx/Mw_/qJI ?xI
/b_h/Lhx/Mw_/Fnb GVp
/b_h/Lhx/Mw_/Y79 8*n
/b_h/JTk &gH
/b_h/JTk/cUW d`h
/b_h/JTk/cUW/7r7 %v>
/b_h/JTk/cUW/Mlj Dgo
/b_h/JTk/cUW/Le5 re[
/b_h/JTk/GLC ];*
/b_h/JTk/GLC/KwL ;2X
/b_h/JTk/GLC/94G uGt
/b_h/JTk/GLC/nHn "7|
/b_h/JTk/ba0 y,]
/b_h/JTk/ba0/qoU Tm[
/b_h/JTk/ba0/RKr mNs
/b_h/JTk/ba0/Cbn 3V{
/b_h/BOs $Qh
/b_h/BOs/oBC 15f
/b_h/BOs/oBC/5t_ /N@
/b_h/BOs/oBC/DwS K9J
/b_h/BOs/oBC/RSp 2Y{
/b_h/BOs/TPq [b}
/b_h/BOs/TPq/jX2 @=D
/b_h/BOs/TPq/nN0 0)t
/b_h/BOs/TPq/qLs J {
/b_h/BOs/RKv p7e
/b_h/BOs/RKv/8jF O~d
/b_h/BOs/RKv/ZDB >a<
/b_h/BOs/RKv/pCN !|O
/zgN E0X
/zgN/HJf }a*
/zgN/HJf/UHb j7P
/zgN/HJf/UHb/CWR xoI
/zgN/HJf/UHb/cUl 5EA
/zgN/HJf/UHb/6eu %Tq
/zgN/HJf/wFU tnt
/zgN/HJf/wFU/q_q &zf
/zgN/HJf/wFU/wbK w|+
/zgN/HJf/wFU/4P8 wqx
/zgN/HJf/dzx ~,<
/zgN/HJf/dzx/8V2 }/z
/zgN/HJf/dzx/xgo XY3
/zgN/HJf/dzx/NEB !#\
/zgN/AFK ]|5
/zgN/AFK/M4K M'J
/zgN/AFK/M4K/Hn3 T@%
/zgN/AFK/M4K/YlE D`r
/zgN/AFK/M4K/j1e 8o$
/zgN/AFK/aXE XgQ
/zgN/AFK/aXE/jBv d&@
/zgN/AFK/aXE/Bad 4)D
/zgN/AFK/aXE/jDv L1q
/zgN/AFK/33m ?6E
/zgN/AFK/33m/iBj 0>/
/zgN/AFK/33m/hbg f3R
/zgN/AFK/33m/Ls3 _et
/zgN/y7l NQr
/zgN/y7l/3Eq 1 8
/zgN/y7l/3Eq/9KV *L=
/zgN/y7l/3Eq/UDg Pm*
/zgN/y7l/3Eq/_z7 )XL
/zgN/y7l/roY wIR
/zgN/y7l/roY/xEX =@#
/zgN/y7l/roY/YzW `?J
/zgN/y7l/roY/CQL OV@
/zgN/y7l/tZu DS\
/zgN/y7l/tZu/T_Q }A"
/zgN/y7l/tZu/ivU ^i/
/zgN/y7l/tZu/p3Z "?H
